rules/naming-convention.js

```diff
@@ -47,4 +59,10 @@ module.exports = [
     format: ['PascalCase'],
     custom: { regex: '^I[A-Z]', match: false },
   },
+  {
+    selector: 'variable',
+    types: ['boolean'],
+    format: ['PascalCase'],
+    prefix: ['is', 'should', 'has', 'can', 'did', 'will', 'enable', 'loading'],
+  },
```

Comment:
We can't have it here, as it needs type-checking, we already have it in the `type-checking.js` file